src/org/thoughtcrime/securesms/search/SearchFragment.java

```diff
@@ -68,8 +73,8 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     this.locale = (Locale) getArguments().getSerializable(EXTRA_LOCALE);
 
-    viewModel = ViewModelProviders.of(this, (ViewModelProvider.Factory) new SearchViewModel.Factory(getContext())).get(SearchViewModel.class);
-    DcEventCenter eventCenter = DcHelper.getEventCenter(getContext());
+    viewModel = ViewModelProviders.of(this, (ViewModelProvider.Factory) new SearchViewModel.Factory(requireContext())).get(SearchViewModel.class);
+    DcEventCenter eventCenter = DcHelper.getEventCenter(requireContext());
```

Comment:
all this changes from  getContext() to requireContext() were while fixing all warnings from Android Studio so I could be sure I didn't let in something wrong with the changes, ideally I should have added it in a different commit :/